README.md

```diff
@@ -122,8 +122,8 @@ The `hoprd` provides various command-line switches to configure its behaviour. F
 ```sh
 $ hoprd --help
 Options:
-      --environment <ENVIRONMENT>
-          Environment id which the node shall run on [env: HOPRD_ENVIRONMENT=] [possible values: anvil-localhost, master-staging, debug-staging, anvil-localhost2, monte_rosa]
+      --network <NETWORK>
+          Environment id which the node shall run on [env: HOPRD_NETWORK=] [possible values: anvil-localhost, master-staging, debug-staging, anvil-localhost2, monte_rosa]
```

Comment:
Should rename `master-staging` to `master`

Reply:
changed